R/ResampleInstances.R

```diff
@@ -69,3 +69,66 @@ instantiateResampleInstance.SpRepCVDesc = function(desc, size, task = NULL) {
   g = as.factor(rep(seq_len(desc$reps), each = folds))
   makeResampleInstanceInternal(desc, size, train.inds = train.inds, test.inds = test.inds, group = g)
 }
+
+
+instantiateResampleInstance.FixedWindowCVDesc = function(desc, size, task = NULL, coords) {
+  initial.window.abs = scaleWindows(desc$initial.window, size)
```

Comment:
Why isn't the scaled initial window stored in the description object like horizon and skip?

Reply:
I put the other ones in desc for convenience. I can add initial window to change the value in `desc` as well

Reply:
Up to you, but I would make it consistent -- either all or nothing in desc, and if it goes in there I would document that.

Reply:
Word, I just set them both to be copied with meaningful names in the func